example/20-componentconfig-gardenlet.yaml

```diff
@@ -69,7 +69,7 @@ controllers:
     syncPeriod: 30s
   seed:
     concurrentSyncs: 5
-    syncPeriod: 1m
+    syncPeriod: 15m
```

Comment:
Maybe again use `1h` for the local setups unless we have a good reason to use a low syncPeriod such as `15m`?